File: Tutoriels/Cpp/POO/Heriatge_Probleme/src/GPerson.cpp
//===============================================
#include "GPerson.h"
//===============================================
GPerson::GPerson() {
    m_name = "";
    m_email = "";
}
//===============================================
GPerson::GPerson(const string& name, const string& email) {
    m_name = name;
    m_email = email;
}
//===============================================
GPerson::~GPerson() {

}
//===============================================
void GPerson::toString() const {
    string lData = "";
    lData += "Personne: ";
    lData += m_name + " | " + m_email + "\n";
    cout << lData.c_str();
}
//===============================================

File: Tutoriels/Python/POO/Classe_Methode_Statique/src/GPerson.py
#================================================
class GPerson: 
    #================================================
    def __init__(self):
        self.m_name = "Gerard"
        self.m_email = "gerard@python.com"
    #================================================
    def ToString():
        print "Description: Je suis une methode statique"
    #================================================
    ToString = staticmethod(ToString)
#================================================
